app/containers/App/Application.js

```diff
@@ -81,13 +81,18 @@ import TeamAdminMessages from "../MassEnergizeSuperAdmin/Messages/TeamAdminMessa
 import TeamMembers from "../MassEnergizeSuperAdmin/Teams/TeamMembers";
 import EventRSVPs from "../MassEnergizeSuperAdmin/Events/EventRSVPs";
 import ThemeModal from "../../components/Widget/ThemeModal";
-
+import { apiCall, PERMISSION_DENIED } from "../../utils/messenger";
+const THIRTY_MINUTES = 1000 * 60 * 30;
+const TWENTY_FOUR_HOURS = 1000 * 60 * 60 * 24;
 class Application extends React.Component {
   componentWillMount() {
     this.props.reduxCallCommunities();
   }
   componentDidMount() {
     this.props.fetchInitialContent(this.props.auth);
+    setTimeout(() => {
+      this.runAdminStatusCheck();
+    }, TWENTY_FOUR_HOURS);
```

Comment:
add a comprehensive comment on what is happening here.  Similar to what you added in the PR description